src/sdk/model/vm/vm.ts

```diff
@@ -1295,6 +1295,23 @@ export class Vm extends Entity implements EntityWithPerfSamples {
       return new Blob([response.data], { type: 'application/zip' });
     });
   }
+
+  /**
+   * Get an Advanced Backups download bundle link
+   * @param taskUuid {string} the task uuid used to create bundle
+   * @param fileName {string} optional file name
+   * @returns {Observable<string>}
+   */
+  /* istanbul ignore next: autogenerated */
+  getDownloadBundleLink(taskUuid: string, fileName?: string): Observable<string> {
```

Comment:
Also, would it be possible to pass the token from the Console in these params, so a subscription wouldn't be necessary? Or would that be weird with 3rd parties using the SDK? Or some other token expiration issues?

Reply:
The token is kept sdk side not console side so calling the sdk to get the token to then pass it in to another sdk method doesn't make sense. And using the observable verifies we'll get the most up to date token, the other method getTokenSync could essentially return undefined